mmv1/third_party/terraform/website/docs/r/storage_bucket.html.markdown

```diff
@@ -269,6 +285,11 @@ The following arguments are supported:
 
 * `effective_time` - (Computed) Server-determined value that indicates the time from which the policy, or one with a greater retention, was effective. This value is in RFC 3339 format.
 
+<a name="nested_hierarchical_namespace"></a>The `hierarchical_namespace` block supports:
+
+* `enabled` - (Optional) The flag enables bucket folder management operations provide reliability and management capabilities, including creating, deleting, listing, and renaming folders. 
```

Comment:
This can be improved by breaking a long sentence into smaller and more clear sentences.